api/v1beta1/disruption_types.go

```diff
@@ -198,16 +198,38 @@ func (dd DisruptionDuration) Duration() time.Duration {
 }
 
 type TargetInjection struct {
-	InjectorPodName string `json:"injectorPodName,omitempty"`
+	InjectorPodName    string `json:"injectorPodName,omitempty"`
+	DisruptionKindName string `json:"disruptionKindName,omitempty"`
 	// +kubebuilder:validation:Enum=NotInjected;Injected;IsStuckOnRemoval
 	// +ddmark:validation:Enum=NotInjected;Injected;IsStuckOnRemoval
 	InjectionStatus chaostypes.DisruptionTargetInjectionStatus `json:"injectionStatus,omitempty"`
 	// since when this status is in place
 	Since metav1.Time `json:"since,omitempty"`
 }
 
+type TargetInjectorList []TargetInjection
+
 // TargetInjections map of target injection
-type TargetInjections map[string]TargetInjection
+type TargetInjections map[string]TargetInjectorList
```

Comment:
What do you think of making this `map[string]map[string]TargetInjection?` Or alternatively, keeping this but changing `type TargetInjectorList` into `type TargetInjectorMap map[DisruptionKind]TargetInjection` in order to change the kubectl describe output to be a little nicer and separate the fields from separate injections

Reply:
I agree with you, it will be less complex to parse 👍